digitalpy/core/zmanager/integration_manager.py

```diff
@@ -0,0 +1,43 @@
+################
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D100 Missing docstring in public module